source/plugins/go/src/go.mod

```diff
@@ -1,81 +1,34 @@
 module Docker-Provider/source/plugins/go/src
 
-go 1.21
+go 1.14
 
 require (
-	github.com/Azure/azure-kusto-go v0.15.1
+	github.com/Azure/azure-kusto-go v0.5.2
+	github.com/Azure/go-autorest/autorest v0.11.28 // indirect
+	github.com/Azure/go-autorest/autorest/adal v0.9.22 // indirect
+	github.com/Azure/go-autorest/autorest/azure/auth v0.5.12
```

Comment:
are you just reverting changes back to  before working  state ? or are there any changes on top of working state?

Reply:
No just back to the working state from before